src/current/_includes/releases/v24.2/v24.2.0-rc.1.md

```diff
@@ -0,0 +1,37 @@
+## v24.2.0-rc.1
+
+Release Date: August 6, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-0-rc-1-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Added a new Kafka [changefeed sink]({% link v24.2/changefeed-sinks.md %}) that uses the [`franz-go` library](https://github.com/twmb/franz-go) and CockroachDB's `batching_sink` implementation. The new Kafka sink can be enabled with the [`changefeed.new_kafka_sink_enabled`]({% link v24.2/cluster-settings.md %}) cluster setting, which is disabled by default. [#127899][#127899]
+- The v2 Kafka [changefeed sink]({% link v24.2/changefeed-sinks.md %}) now supports [Amazon Managed Streaming for Apache Kafka (MSK)](https://aws.amazon.com/msk/) IAM SASL authentication. [#127899][#127899]
+
+<h3 id="v24-2-0-rc-1-db-console-changes">DB Console changes</h3>
+
+- The [Databases]({% link v24.2/ui-databases-page.md %}) and [Tables]({% link v24.2/ui-databases-page.md %}#tables-view) pages in the [DB Console]({% link v24.2/ui-overview.md %}) will show a loading state while loading information for databases and tables including size and range counts. [#127696][#127696]
+- On the [Database details]({% link v24.2/ui-databases-page.md %}) page, the table name will no longer appear with quotes around the schema and table name. [#127770][#127770]
+
+<h3 id="v24-2-0-rc-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused a memory leak when executing SQL statements with [comments]({% link v24.2/comment-on.md %}), for example, `SELECT /* comment */ 1;`. Memory owned by a SQL session would continue to grow as these types of statements were executed. The memory would only be released when closing the [SQL session]({% link v24.2/cockroach-sql.md %}). This bug has been present since v23.1. [#127760][#127760]
```

Comment:
the type of comment being mentioned here is not the `COMMENT ON` SQL statement

it refers to inline comments in the actual SQL code

we don't have any docs on that, so I'd suggest removing the link and changing 'comments' to 'inline comments' 

I think a better link for 'SQL session' is https://cockroachlabs.com/docs/v24.2/show-sessions since not all sessions are created by the `cockroach sql` binary

Reply:
(at least I don't think we have docs on it)